internal/pkg/cli/pipeline_init.go

```diff
@@ -225,6 +242,51 @@ func (o *initPipelineOpts) validateURL(url string) error {
 	return nil
 }
 
+func (o *initPipelineOpts) validateURLExists(url string) error {
+	repos, err := o.fetchAndParseURLs()
+	if err != nil {
+		return fmt.Errorf("fetch and parse URLs: %w", err)
+	}
+	if _, ok := repos[url]; !ok {
+		return fmt.Errorf("URL '%s' is not a local git remote; please check that you're in the correct directory", o.repoURL)
```

Comment:
I feel like this is more like a friendly message that could be put into a `log.Errorf` block, for error, maybe we could return something like `fmt.Errorf("url %s not found in git remote", o.repoURL)`

Reply:
true, true. I'll add to docs as well.

Reply:
The same could be said of this https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/cli/pipeline_init.go#L339